### DISCUSSION TITLE
Signal-messenger integration

### DISCUSSION THREAD

**Discussion by osm-frasch** [42wim/matterbridge#1797] [New bridges suggestions]
Hello , short question in the round. 
Is it planned or even possible to integrate/bridge the signal messenger into Matterbridge? 

The signal messenger is becoming more and more popular, is open source, enjoys the highest level of trust, end-to-end encryption, easy to use and offers almost the same features as Whatsapp

It would be great if this could be done

**Comment by 42wim** [comment]
Not planned, PR's welcome.
See https://github.com/42wim/matterbridge/issues/28 for more info.

**Comment by alerque** [comment]
Personally I'd rather see platform specific support tracked in separate issues per platform, but since Signal is lumped in with the "everything else" ticket, see the latest relevant bits in

* [this comment](https://github.com/42wim/matterbridge/issues/28#issuecomment-780920359) about signald
* [this comment](https://github.com/42wim/matterbridge/issues/28#issuecomment-787508330) about a WIP
* [this comment](https://github.com/42wim/matterbridge/issues/28#issuecomment-1090366025) about current library licensing and development status

**Comment by aghand0ur** [comment]
any update on this matter?
